DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/vlan/test_vlan_packets.py

```diff
@@ -0,0 +1,93 @@
+import asyncio
+
+import pytest
+from dent_os_testbed.utils.test_utils.tgen_utils import (tgen_utils_get_dent_devices_with_tgen,
+                                                         tgen_utils_setup_streams, tgen_utils_start_traffic,
+                                                         tgen_utils_stop_traffic, tgen_utils_get_traffic_stats,
+                                                         tgen_utils_dev_groups_from_config,
+                                                         tgen_utils_traffic_generator_connect)
+from dent_os_testbed.utils.test_utils.br_utils import (configure_bridge_setup, configure_vlan_setup)
+
+pytestmark = [pytest.mark.suite_functional_vlan,
+              pytest.mark.asyncio,
+              pytest.mark.usefixtures("cleanup_bridges", "cleanup_tgen")]
+
+port_map = ({"port": 0, "settings": [{"vlan": 1, "untagged": True, "pvid": True}]},
+            {"port": 1, "settings": [{"vlan": 1, "untagged": True, "pvid": True}]},
+            {"port": 2, "settings": [{"vlan": 1, "untagged": True, "pvid": True}]},
+            {"port": 3, "settings": [{"vlan": 1, "untagged": False, "pvid": True}]},)
+
+
+async def test_vlan_tagged_packet_size(testbed):
+    """
+    Test Name:VLAN tagged packet size
+    Test Suite: suite_functional_vlan
+    Test Overview: Test packet with VLAN tag on untagged port is bigger by 4 bytes
```

Comment:
How can we observe tagged packets on an untagged port? Please correct this statement.